### PR TITLE
Add Exercise05

### PR DESCRIPTION
**Pull request by huynhngocminh2511**

### CODE REVIEW

File: Exercise05/Models/SjcGoldPrice.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+        [XmlAttribute(AttributeName = "type")]
+        public string Type { get; set; }
+    }
+
+    [XmlRoot(ElementName = "city")]
+    public class City
+    {
+        [XmlElement(ElementName = "item")]
+        public List<GoldPrice> GoldPrice { get; set; }
```
Comment (voquanghoa) [inline Exercise05/Models/SjcGoldPrice.cs]: Rename `GoldPrice` to `GoldPrices`

File: Exercise05/Models/SjcGoldPrice.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+        [XmlAttribute(AttributeName = "name")]
+        public string Name { get; set; }
+    }
+
+    [XmlRoot(ElementName = "ratelist")]
+    public class Ratelist
+    {
+        [XmlElement(ElementName = "city")]
+        public List<City> City { get; set; }
```
Comment (voquanghoa) [inline Exercise05/Models/SjcGoldPrice.cs]: Rename `City` to `Cities`

File: Exercise05/MainActivity.cs [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+            var refresh = FindViewById<SwipeRefreshLayout>(Resource.Id.srl_refresh);
+            refresh.Refresh += delegate
+            {
+                ReLoad();
+                refresh.Refreshing = false;
+            };
+        }
+
+        public void ReLoad()
```
Comment (voquanghoa) [inline Exercise05/MainActivity.cs]: Bad name. It can be `LoadGoldData`

File: Exercise05/Api/SjcService.cs [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        }
+
+        private List<string> strings;
+
+        public List<string> Strings
+        {
+            get => strings;
+        }
+
+        public void ReloadSjcGoldPrice()
```
Comment (voquanghoa) [inline Exercise05/Api/SjcService.cs]: Bad name.

File: Exercise05/Api/SjcService.cs [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+        private List<City> cities;
+
+        public List<City> Cities
+        {
+            get => cities;
+        }
+
+        private List<string> strings;
+
+        public List<string> Strings
```
Comment (voquanghoa) [inline Exercise05/Api/SjcService.cs]: Make it easy
```C#
public List<string> Strings{get;}
```

Bad name actually.

File: Exercise05/Api/SjcService.cs [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+{
+    class SjcService
+    {
+        public static SjcService api = new SjcService();
+
+        private string root = "http://www.sjc.com.vn/xml/tygiavang.xml";
+
+        private List<City> cities;
+
+        public List<City> Cities
```
Comment (voquanghoa) [inline Exercise05/Api/SjcService.cs]: `public List<City> Cities{get;}`

File: Exercise05/Models/SjcGoldPrice.cs (hunk 2 above)
Comment (huynhngocminh2511) [reply Exercise05/Models/SjcGoldPrice.cs]: Done

File: Exercise05/Models/SjcGoldPrice.cs (hunk 1 above)
Comment (huynhngocminh2511) [reply Exercise05/Models/SjcGoldPrice.cs]: Done

File: Exercise05/MainActivity.cs (hunk 3 above)
Comment (huynhngocminh2511) [reply Exercise05/MainActivity.cs]: Done

File: Exercise05/Api/SjcService.cs (hunk 4 above)
Comment (huynhngocminh2511) [reply Exercise05/Api/SjcService.cs]: Done

File: Exercise05/Api/SjcService.cs (hunk 5 above)
Comment (huynhngocminh2511) [reply Exercise05/Api/SjcService.cs]: Done

File: Exercise05/Api/SjcService.cs (hunk 6 above)
Comment (huynhngocminh2511) [reply Exercise05/Api/SjcService.cs]: Done

**Pull request merged**